Torch, Raiders and Skrell scoutship no longer have access to thermals

Imienny1 opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

:cl: Imienny
rscdel: Its no longer possible to make combat mech sensors(mech sensors with thermal vision)
rscdel: Skrell scout ship no longer spawns with thermal goggles
tweak: Raiders cybersuit now spawns with mesons
/:cl:

CrimsonShrike committed:
Fixes some exosuit feedback issues

Spookerton committed:
removes some unused code, GLOBifies some globals

SierraKomodo committed:
Add teleporter to example map

SierraKomodo committed:
Update teleporter sprites

Sprites provided by https://github.com/Pawnp

Imienny1 committed:
binoculars bugfixes

Spookerton committed:
update various 0/1 truths to TRUE/FALSE + cleaning

wall_mounted

simulated\s*=\s*.+$ => simulated = TRUE|FALSE

can_self_harvest

misc robot truthy stuff and comment bits

remove unused weapon lock & killswitch from robots

also some leftover vars

wiresexposed

emagged

leaves simple bots and barriers alone for now because they have funky behavior

fabricator emag behavior

tests

Spookerton committed:
security barrier revamp

Spookerton committed:
ram horns are too big - can't use them with ears

Commit:
Firelock improvements

Spookerton committed:
squashéd

CrimsonShrike committed:
Apply suggestions from code review

Co-authored-by: Sierra Brown (SierraKomodo) <SierraKomodo@users.noreply.github.com>

CrimsonShrike committed:
Adds automatic plasma cutter as exosuit gear


Update code/modules/mechs/equipment/utility.dm

Co-authored-by: Sierra Brown (SierraKomodo) <SierraKomodo@users.noreply.github.com>

Spookerton committed:
remove & replace funky high power pump types

Ryan180602 committed:
Removes smell feedback for synthetics



space base

Spookerton committed:
you can't succumb, ghost instead coward

juxjux9930 committed:
gives robotics a paint sprayer

Imienny1 committed:
Thermal Removal

Imienny1 committed:
Thermal Removal

Lorwp committed:
Make actions used in workflows use the SHA-1

per <https://docs.github.com/es/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions>

Commit:
tweak send_to_department() to handle all handhelds

SierraKomodo committed:
Merge pull request #30292 from juxtaposed9930/norlangoaway

Removes lack of robotics paint sprayer

PsiOmegaDelta committed:
Automatic changelog generation for PR #30292 [ci skip]

MuckerMayhem committed:
buffs to some simple mobs damage

PsiOmegaDelta committed:
Automatic changelog update

CrimsonShrike committed:
Tries to untangle carbon damage

Spookerton committed:
add partial rgb2num for <514

beware- does not implement color space argument because we only use
rgb web color strings anyway.

HeyBanditoz committed:
Prevent borg/drone grippers from grabbing modules

Fixes #30221
Fixes #26667

Spookerton committed:
Merge pull request #30295 from Lorwp/actions-useCommitSHA

Make actions used in workflows use the SHA-1 hash instead

PsiOmegaDelta committed:
Automatic changelog update

Lorwp committed:
Voleyball > Volleyball

speedmerge pls thx bye

SierraKomodo committed:
Merge pull request #30311 from Lorwp/voleyball

Voleyball > Volleyball

PsiOmegaDelta committed:
Automatic changelog generation for PR #30311 [ci skip]

CrimsonShrike committed:
Exosuits no longer need full reset for emp damage

SierraKomodo committed:
Fix teleport_far breaking the player_levels global

PsiOmegaDelta committed:
Automatic changelog update

Spookerton committed:
Merge pull request #30314 from SierraKomodo/fix-broken-player-levels

Fix teleport_far breaking the player_levels global

PsiOmegaDelta committed:
Automatic changelog generation for PR #30314 [ci skip]

Spookerton committed:
Merge pull request #30235 from CrimsonShrike/FeedBackTheWhat

Fixes some exosuit feedback issues

Spookerton committed:
Merge pull request #30308 from HeyBanditoz/borg_grip_got_drip

Prevent borg/drone grippers from grabbing modules

Spookerton committed:
Merge pull request #30299 from MuckerMayhem/simplemobs-buff

buffs to some simple mobs damage

PsiOmegaDelta committed:
Automatic changelog generation for PR #30299 [ci skip]

Spookerton committed:
Merge pull request #30284 from CrimsonShrike/AutoPlasma

Adds automatic plasma cutter as exosuit gear

PsiOmegaDelta committed:
Automatic changelog generation for PR #30284 [ci skip]

CrimsonShrike committed:
Open cockpit exosuits no longer move air

SierraKomodo committed:
Add firer to hostile simple mob projectiles

Resolves attack logs showing 'UNKNOWN SUBJECT' from simple mob attacks

PsiOmegaDelta committed:
Automatic changelog update

SierraKomodo committed:
Merge pull request #30296 from Parazoid/pager-tweak

tweak send_to_department() to handle all handheld devices

PsiOmegaDelta committed:
Automatic changelog generation for PR #30296 [ci skip]

SierraKomodo committed:
Merge pull request #30240 from Spookerton/spkrtn/cng/checker-bits-n-pieces

Removes some returns in spawns that cause dreamchecker warnings

SierraKomodo committed:
Merge pull request #30264 from Spookerton/spkrtn/cng/just-a-language-barrier

Security barrier revamp

PsiOmegaDelta committed:
Automatic changelog generation for PR #30264 [ci skip]

SierraKomodo committed:
Merge pull request #30267 from Spookerton/spkrtn/cng/ram-ears

ram horns are too big - can't use them with ears

PsiOmegaDelta committed:
Automatic changelog generation for PR #30267 [ci skip]

PsiOmegaDelta committed:
Automatic changelog update

Commit:
lowercases some names

GinjaNinja32 committed:
Fix throttle check direction to throttle

Now checks if $unthrottleTime is still in the future, rather than if it's in the past.

Spookerton committed:
Merge pull request #30319 from GinjaNinja32/patch-1

Fix throttle check direction to throttle

Spookerton committed:
clears world topic throttle list if in the past

Spookerton committed:
Merge pull request #30272 from Ryan180602/nosmellobot

Synthetics are no longer smelly

PsiOmegaDelta committed:
Automatic changelog generation for PR #30272 [ci skip]

Spookerton committed:
Merge pull request #30269 from Ilysen/firedoor-go-brr

Firedoor code improvements, firedoors now play sounds on open/close

PsiOmegaDelta committed:
Automatic changelog generation for PR #30269 [ci skip]

Spookerton committed:
Merge pull request #30318 from SierraKomodo/ranged-mob-attack-log-fix

Add firer to hostile simple mob projectiles

PsiOmegaDelta committed:
Automatic changelog generation for PR #30318 [ci skip]

Spookerton committed:
Merge pull request #30316 from CrimsonShrike/AiringOut

Open cockpit exosuits no longer move air

Spookerton committed:
Merge pull request #30313 from CrimsonShrike/ExoEndurance

Exosuits no longer need full reset for emp damage

Spookerton committed:
Merge pull request #30320 from Ilysen/lowercasin

lowercases some names

PsiOmegaDelta committed:
Automatic changelog generation for PR #30320 [ci skip]

Spookerton commented on code/modules/clothing/spacesuits/rig/suits/light.dm:
```diff
@@ -85,6 +85,16 @@
 	item_flags = ITEM_FLAG_THICKMATERIAL | ITEM_FLAG_NOCUFFS
 
+/obj/item/weapon/rig/light/hacker/meson
+		initial_modules = list(
+		/obj/item/rig_module/ai_container,
+		/obj/item/rig_module/power_sink,
+		/obj/item/rig_module/datajack,
+		/obj/item/rig_module/electrowarfare_suite,
+		/obj/item/rig_module/voice,
+		/obj/item/rig_module/cooling_unit,
+		/obj/item/rig_module/vision/meson,
+		)
```
```suggestion
		initial_modules = list(
			/obj/item/rig_module/ai_container,
			/obj/item/rig_module/power_sink,
			/obj/item/rig_module/datajack,
			/obj/item/rig_module/electrowarfare_suite,
			/obj/item/rig_module/voice,
			/obj/item/rig_module/cooling_unit,
			/obj/item/rig_module/vision/meson
		)
```

EvenInDeathIStillServe committed:
Removes lack of RUST coloration

Commit:
cable coil examine standardization

Spookerton committed:
Merge pull request #30322 from Spookerton/spkrtn/fix/shine-a-light-on-me

clears world topic throttle list if in the past

Spookerton committed:
add map & site player cost

Spookerton committed:
Merge pull request #30255 from SierraKomodo/teleporter-sprites

Teleporter sprites

PsiOmegaDelta committed:
Automatic changelog generation for PR #30255 [ci skip]

Spookerton committed:
Removes base type storage accessory from raiders

SierraKomodo committed:
Merge pull request #30323 from Ilysen/coral

cable coil examine text standardization

PsiOmegaDelta committed:
Automatic changelog generation for PR #30323 [ci skip]

Spookerton committed:
Merge pull request #30325 from Spookerton/spkrtn/fix/hold-this-for-me

Removes base type storage accessory from raiders

SierraKomodo committed:
Merge pull request #30306 from Spookerton/spkrtn/cng/pink-fluffy-unicorns

add partial rgb2num for <514

juxjux9930 committed:
Steel and plasteel are stronk

PsiOmegaDelta committed:
Automatic changelog update

juxjux9930 committed:
Reinforcen't

SierraKomodo committed:
Merge pull request #30326 from juxtaposed9930/steelgohnnngh

Steel and plasteel are stronk

PsiOmegaDelta committed:
Automatic changelog generation for PR #30326 [ci skip]

CrimsonShrike committed:
Catwalks are magnetic too, dude

SierraKomodo committed:
Merge pull request #30290 from Spookerton/spkrtn/cng/pump-it

remove & replace funky high power pump types

SierraKomodo committed:
Merge pull request #30291 from Spookerton/spkrtn/cng/no-surrender

Succumb verb is gone

PsiOmegaDelta committed:
Automatic changelog generation for PR #30291 [ci skip]

SierraKomodo committed:
Merge pull request #30279 from EvenInDeathIStillServe/dasrust

Removes lack of RUST coloration

PsiOmegaDelta committed:
Automatic changelog generation for PR #30279 [ci skip]

PsiOmegaDelta committed:
Automatic changelog update

Imienny1 committed:
Update code/modules/clothing/spacesuits/rig/suits/light.dm

Co-authored-by: Spookerton <spkrtn@pm.me>

PsiOmegaDelta committed:
Automatic changelog update

MuckerMayhem committed:
artifact updates and fixes

Tweaks pushback so it doesn't tear off limbs, fixed the analyzer not detecting secondary effects, removed gas effects, added toggle to artifact effects when triggered so they stay on longer.

SierraKomodo committed:
Merge pull request #30289 from Spookerton/spkrtn/sys/player-costs

Add player threshold for sites with slots & cost per site

SierraKomodo committed:
Merge pull request #30304 from CrimsonShrike/StunLockHowAboutStunNo

Tries to untangle carbon damage

PsiOmegaDelta committed:
Automatic changelog update

PsiOmegaDelta committed:
Automatic changelog update

EvenInDeathIStillServe committed:
Fixes bug with loading crossbows

SierraKomodo committed:
Merge pull request #30329 from CrimsonShrike/MagneticCats

Catwalks are magnetic too, dude

SierraKomodo committed:
Merge pull request #30265 from Spookerton/spkrtn/cng/more-misc-truthies

More truth

PsiOmegaDelta committed:
Automatic changelog generation for PR #30265 [ci skip]

SierraKomodo committed:
Merge pull request #30236 from Spookerton/spkrtn/cng/misc-code-&-globals

Removes some naked globals & an unused admin proc

SierraKomodo committed:
Merge pull request #30340 from EvenInDeathIStillServe/das_crossbow

Fixes bug with loading crossbows

PsiOmegaDelta committed:
Automatic changelog update

Spookerton committed:
obj/item/weapon/ -> obj/item/ + post

PsiOmegaDelta committed:
Merge pull request #30350 from Spookerton/spkrtn/cng/objectify-me

Removes /weapon/

PsiOmegaDelta committed:
Automatic changelog update

SierraKomodo committed:
Merge pull request #30298 from MuckerMayhem/artifact-changes

QOL Artifact changes

PsiOmegaDelta committed:
Automatic changelog generation for PR #30298 [ci skip]

SierraKomodo committed:
Merge pull request #30327 from juxtaposed9930/reinforcen't

Reinforcen't

PsiOmegaDelta committed:
Automatic changelog generation for PR #30327 [ci skip]

SierraKomodo committed:
Merge pull request #30256 from Imienny1/ThermalRevert

Fixes few binoculars/scopes bugs

PsiOmegaDelta committed:
Automatic changelog generation for PR #30256 [ci skip]

SierraKomodo reviewed (commented):
With #30256, this probably isn't necessary anymore.

SierraKomodo reviewed (changes requested):
Also, merge conflicts

Imienny1 committed:
resolve merge conflicts(hopefully) and give raiders back normal hacker rig

Imienny1 committed:
resolved merge conflicts(hopefully) and give raiders back normal hacker rig
